corpus/stats.py

```diff
@@ -16,24 +16,44 @@ class ExtractOovWordsFromCorpusJob(Job):
     Extracts the out of vocabulary words based on a given corpus and lexicon
     """
 
-    def __init__(self, bliss_corpus, bliss_lexicon):
+    __sis_hash_exclude__ = {
+        "casing": "upper",
+    }
+
+    def __init__(self, bliss_corpus, bliss_lexicon, casing="none"):
```

Comment:
Why is default "none" but sis_hash_exclude has "upper"? This will break old setups.

Reply:
yes. I missed that.

When I think about this.. the old behavior was upper. so the default value and sis_hash_exclude should be "upper" correct?
@JackTemaki you agree?

Reply:
but we set this in the Librispeech pipeline to "upper".. and that for that case the hash should not change. but for every new setup we want the "none" behavior.. not sure how to handle this...

Reply:
But would it not be correct to have "upper" as default, and exclude this from the hash? Then it does not change anything anywhere.

Reply:
>  but for every new setup we want the "none" behavior
New behavior newer works without breaking old stuff, this is why RETURNN got this `behavior_version` parameter...

Reply:
What if we add a "second" version of the job with changed defaults...? I have no better idea....

Reply:
It would inherit completely from the first one except for that single change

Reply:
maybe have "upper" as default but add a warning for command line or log file!?

Reply:
"upper" is now default and included in sis_hash_exclude to not break legacy settings. but I added a warning

Reply:
maybe move the warning to the run function??